pkg/apis/rollouts/v1alpha1/types.go

```diff
@@ -248,6 +248,14 @@ type CanaryStep struct {
 	Analysis *RolloutAnalysisStep `json:"analysis,omitempty"`
 }
 
+// RolloutAnalysisBackground defines a template that is used to create a background analysisRun
+type RolloutAnalysisBackground struct {
+	RolloutAnalysisStep `json:",inline"`
+	// StartAtStep indicates which step the background analysis should start on
+	// If not listed, controller defaults to 0
+	StartAtStep *int32 `json:"startAtStep,omitempty"`
```

Comment:
What do you think of:`startingStep` ? Might be easier to talk about it conversationally.